### PR TITLE
Userprofile

### PR DESCRIPTION
**Pull request by tushartiwari7**
added data recieved from api calls of BE to redux store and used that to render profile page.

### CODE REVIEW

**Review by Bhakti199 — APPROVED** [review]
Good to merge, no issues found.

File: src/app/features/userSlice.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	reducers: {},
+	extraReducers: {
+		[getAllUsers.fulfilled]: (state: any, action: any) => {
+			console.log({action});
+			state = action.payload;
+			return action.payload;
+		}
+	},
+});
+
```
Comment (JantuDeb) [inline src/app/features/userSlice.ts:29]: You can try to add an interface for the state instead of using any

**Review by JantuDeb — APPROVED** [review]
Overall the code looks good

**Review by ashishnnnnn — APPROVED** [review]
Code looks good to merge..
Liked the use of createAsyncThunk with redux..

File: src/app/features/userSlice.ts (hunk 1 above)
Comment (tushartiwari7) [reply src/app/features/userSlice.ts:29]: Yes, will do one refactor for that after some time. Thanks.

**Pull request merged**